### PR TITLE
feat(pci.db): support hour month toggle in pages displaying prices

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-189 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

 Add toggle to display pricing in hour or month in very pages customer can purchase something in database.

### 🎏 Translations
- [x] TRANS-52565

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -4,5 +4,7 @@
   "pci_databases_general_information_add_node_cancel": "Annuler",
   "pci_databases_general_information_add_node_description": "Vous disposez actuellement de {{nodesCount}} nœud(s). Voulez-vous vraiment en ajouter un ? Votre facture sera augmentée de",
   "pci_databases_general_information_add_node_success": "Votre demande d’ajout de nœud a bien été prise en compte et sera effective dans quelques minutes.",
-  "pci_databases_general_information_add_node_error": "Une erreur est survenue lors d'ajout de nœud : {{ message }}"
+  "pci_databases_general_information_add_node_error": "Une erreur est survenue lors d'ajout de nœud : {{ message }}",
+  "pci_databases_general_information_price_unit_month": "/ mois",
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_fr_FR.json:8]:
```suggestion
  "pci_databases_general_information_price_unit_month": "/mois",
```
French translation should not display "spaces" around slashes.

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -4,5 +4,7 @@
...
+  "pci_databases_general_information_price_unit_hour": "/ heure"
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_fr_FR.json:9]:
```suggestion
  "pci_databases_general_information_price_unit_hour": "/heure"
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/add-node.controller.js [hunk 3]
```diff
@@ -21,18 +22,28 @@ export default class {
...
+      ? Math.max(
+          0,
+          addedStorageByNode *
+            this.currentFlavor.hourlyPricePerGB.priceInUcents,
+        )
+      : Math.max(
+          0,
+          addedStorageByNode *
+            this.currentFlavor.monthlyPricePerGB.priceInUcents,
+        );
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/add-node.controller.js:45]: Maybe this is a bit more readable:
```suggestion
  const additionalStoragePrice = Math.max(
  0, 
  this.showMonthlyPrices
    ? addedStorageByNode * this.currentFlavor.hourlyPricePerGB.priceInUcents
    : addedStorageByNode * this.currentFlavor.monthlyPricePerGB.priceInUcents
```

Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/add-node.controller.js]: Hello, yes you are right, thx! Fixed

**Review by anooparveti — DISMISSED** [review]

**Review by jperchoc — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by jperchoc — APPROVED** [review]

**Pull request merged**